packages/fuels/src/cli/commands/build/buildSwayProgram.ts

```diff
@@ -10,11 +9,8 @@ export const buildSwayProgram = async (config: FuelsConfig, path: string) => {
   debug('Building Sway program', path);
 
   return new Promise<void>((resolve, reject) => {
-    const builtInForcPath = findBinPath('fuels-forc', __dirname);
-
-    const command = config.useBuiltinForc ? builtInForcPath : 'forc';
     const args = ['build', '-p', path].concat(config.forcBuildFlags);
-    const forc = spawn(command, args, { stdio: 'pipe' });
+    const forc = spawn('forc', args, { stdio: 'pipe' });
```

Comment:
Just exemplifying how [this](https://github.com/FuelLabs/fuels-ts/pull/2143/files#r1576798648) suggestion could work:

```suggestion
    const forc = spawn(config.forcPath ?? 'forc', args, { stdio: 'pipe' });
```